owid/importers
# Update VDEM dataset to 11.1

- Update code for v11.1
- Always download latest data
- VDEM v11.1
- Create __init__.py
- Update link
- Fix quotes
- Fix source description quotes
- Fix notes -> description
- Fix naming conflict
- Fix typo
- Revert "Fix typo"
- Revert "Fix naming conflict"
- Check that sources file has its own set of ids
- Update sources.csv
- Add id to sources
- Update import_dataset.py
- Fix quotes
- Update distinct_countries_standardized.csv
- Update sources.csv
- Clear previous output
- Set default code to NULL rather than ''
- Clean missing values
- New processing
